easyocr/detection.py

```diff
@@ -45,6 +45,9 @@ def test_net(canvas_size, mag_ratio, net, image, text_threshold, link_threshold,
     with torch.no_grad():
```

Comment:
```suggestion
    with torch.inference_mode():
```

Reply:
Seems useful but perhaps should be a separate PR? I was focused on the GPU memory leak with this PR so not sure if they should be packaged together.